pkg/cloud/aws/regions/regions.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package regions
+
+import "os"
+
+// Default is the region used when no region is specified.
+var Default string
```

Comment:
since this isn't how cluster-api-provider-aws sets regions for sessions could this be kept to the test package? Otherwise I'm concerned this will cause confusion (much like you experienced) when people are reading through this code.

Reply:
Hi @chuckha,

I think that makes sense. I'll update the PR.